### ISSUE TITLE
Fix negative vesting withdrawals

### ISSUE THREAD

**Issue by VIM-Arcange** [GolosChain/golos#810]
Steem network crashed because of a flaw in steem_evaluator.cpp with negative withdrawal

https://github.com/steemit/steem/pull/2583/files

Golos code should be checked updated to avoid similar issue ASAP.

**Comment by inertia186** [comment]
According to [GolosSQL](https://golos.cloud/):

```sql
SELECT [TxWithdraws].* FROM [TxWithdraws] WHERE (vesting_shares < 0)
```

The result is currently zero rows.  So keep an eye on that.

**Comment by afalaleev** [comment]
Fixed in v0.18.3
